system-config/select-apt-mirror.sh

```diff
@@ -20,7 +20,7 @@
 # This script changes the apt mirror to use.
 
 if [[ "$1" -ne "" ]]; then
-    releaseName=$1
+    releaseName="$1"
```

Comment:
I agree, and I pushed a change with a usage line. The script still defaults to testing, which I'm fine with.